app/views/document_associations/associations/_multi_association_input.html.erb

```diff
@@ -0,0 +1,13 @@
+<p>
```

Comment:
We open a `<p>` tag here but I don't see any close? not certain we even need the `<p>` ?